client/lib/geocoding/test/index.js

```diff
@@ -15,6 +16,8 @@ import { geocode } from '../';
 const TEST_ADDRESS = '1600 Amphitheatre Parkway, Mountain View, CA';
 
 describe( 'geocoding', () => {
+	useNock();
```

Comment:
We could improve that a bit and do similar trick we did for `useMockery` and pass `before` callback as first param to `useNock` call. Then wouldn't have to import `nock` explicitly in this file. What do you think?